modules/ROOT/pages/hdfs/hdfs-connector-reference.adoc

```diff
@@ -1,649 +1,603 @@
 |===
 
 == Operations
 
-* <<append>>
-* <<copyFromLocalFile>>
-* <<copyToLocalFile>>
-* <<deleteDirectory>>
-* <<deleteFile>>
-* <<getMetadata>>
-* <<globStatus>>
-* <<listStatus>>
-* <<makeDirectories>>
-* <<readOperation>>
-* <<rename>>
-* <<setOwner>>
-* <<setPermission>>
-* <<write>>
+* <<append>> 
+* <<copyFromLocalFile>> 
+* <<copyToLocalFile>> 
+* <<deleteDirectory>> 
+* <<deleteFile>> 
+* <<getMetadata>> 
+* <<globStatus>> 
+* <<listStatus>> 
+* <<makeDirectories>> 
+* <<readOperation>> 
+* <<rename>> 
+* <<setOwner>> 
+* <<setPermission>> 
+* <<write>> 
 
 === Associated Sources
-* <<read>>
+* <<read>> 
 
 
 [[append]]
 === Append
-
 `<hdfs:append>`
 
 
-Append the current payload to a file located at the designated path. Note: by default the Hadoop server has the append option disabled. To be able append any data to an existing file refer to dfs.support.append configuration parameter.
+Append the current payload to a file located at the designated path. *Note:* by default the Hadoop server has the append option disabled. To append data to an existing file, refer to the dfs.support.append configuration parameter.
 
-[cols=".^20%,.^20%,.^35%,.^20%,^.^5%", options="header"]
+
+==== Parameters
+[%header,cols="20s,20a,35a,20a,5a"]
 |===
 | Name | Type | Description | Default Value | Required
 | Configuration | String | The name of the configuration to use. | | x
-| Path a| String |  the path of the file to write to. |  | x
-| Buffer Size a| Number |  the buffer size to use when appending to the file. |  4096 |
-| Payload a| Binary |  the payload to append to the file. |  `#[payload]` |
+| Path a| String |  The path of the file to write to. |  | x
+| Buffer Size a| Number |  The buffer size to use when appending to the file. |  4096 | 
+| Payload a| Binary |  The payload to append to the file. |  #[payload] | 
 | Reconnection Strategy a| * <<reconnect>>
-* <<reconnect-forever>> |  A retry strategy in case of connectivity errors |  |
+* <<reconnect-forever>> |  A retry strategy in case of connectivity errors |  | 
 |===
 
 
 === For Configurations
+<<hdfs>> 
 
-* <<hdfs>>
-
-=== Throws
+==== Throws
 
-* HDFS:CONNECTIVITY
-* HDFS:INVALID_REQUEST_DATA
-* HDFS:INVALID_STRUCTURE_FOR_INPUT_DATA
-* HDFS:RETRY_EXHAUSTED
-* HDFS:UNKNOWN
+* HDFS:CONNECTIVITY 
+* HDFS:INVALID_REQUEST_DATA 
+* HDFS:INVALID_STRUCTURE_FOR_INPUT_DATA 
+* HDFS:RETRY_EXHAUSTED 
+* HDFS:UNKNOWN 
 
 
 [[copyFromLocalFile]]
-== Copy From Local File
-
+=== Copy From Local File
 `<hdfs:copy-from-local-file>`
 
 
-Copy the source file on the local disk to the FileSystem at the given target path, set deleteSource if the source should be removed.
-
+Copy the source file on the local disk to the file system for a target path, set *Delete Source* if the source should be removed.
```

Comment:
Copy the source file on the local disk to the file system for a target path and set *Delete Source* to remove the source (file???).